### PR TITLE
Arwen v1.4

### PR DESCRIPTION
**Pull request by sasurobert**
Option for arwen v1.4

### CODE REVIEW

File: process/factory/shard/vmContainerFactory.go [hunk 1]
```diff
@@ -38,6 +40,7 @@ type vmContainerFactory struct {
 	container                      process.VirtualMachinesContainer
 	arwenVersions                  []config.ArwenVersionByEpoch
 	arwenChangeLocker              process.Locker
+	eSDTTransferParser             vmcommon.ESDTTransferParser
```
Comment (SebastianMarian) [inline process/factory/shard/vmContainerFactory.go:43]: esdtTransferParser

File: process/factory/shard/vmContainerFactory.go [hunk 2]
```diff
@@ -268,6 +278,23 @@ func (vmf *vmContainerFactory) createInProcessArwenVMV13() (vmcommon.VMExecution
 	return arwenHost13.NewArwenVM(vmf.blockChainHookImpl, hostParameters)
 }
 
+func (vmf *vmContainerFactory) createInProcessArwenVMV14() (vmcommon.VMExecutionHandler, error) {
+
```
Comment (SebastianMarian) [inline process/factory/shard/vmContainerFactory.go:282]: Remove empty line

**Review by iulianpascalau — DISMISSED** [review]
The new Arwen version adds ~1,16MB

File: process/factory/shard/vmContainerFactory.go (hunk 1 above)
Comment (sasurobert) [reply process/factory/shard/vmContainerFactory.go]: done

File: process/factory/shard/vmContainerFactory.go (hunk 2 above)
Comment (sasurobert) [reply process/factory/shard/vmContainerFactory.go]: done

**Review by iulianpascalau — DISMISSED** [review]

**Review by SebastianMarian — APPROVED** [review]

**Review by iulianpascalau — DISMISSED** [review]

**Review by gabi-vuls — APPROVED** [review]
System test passed. 
New error is legitimate.
image

**Review by SebastianMarian — APPROVED** [review]

**Review by iulianpascalau — APPROVED** [review]
Will need further testing on this

**Pull request merged**